lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -185,7 +188,9 @@ export default function BasicLtiLaunchApp({ rpcServer }) {
     }
   }, [authToken, canvas.speedGrader, contentUrl]);
 
-  useEffect(reportSubmission, [reportSubmission]);
+  useEffect(() => {
+    reportSubmission();
+  }, [reportSubmission]);
```

Comment:
This change might look like a no-op, but it ensures that the return value of `reportSubmission` is not passed back `useEffect`, which was not intended.

Reply:
👍